python/lsst/qserv/tests/qservDbLoader.py

```diff
@@ -134,6 +135,7 @@ def dropCssDatabase(self):
 
     def workerInsertXrootdExportPath(self):
         if self.multi_node:
-            self.wmgr.xrootdRegisterDb(self._dbName, allowDuplicate=True)
+            for node in self.nodeNames:
+                self.nWmgrs[node].xrootdRegisterDb(self._dbName, allowDuplicate=True)
```

Comment:
SAme here:

    for wmgr in self.nWmgrs.values():
        wmgr.xrootdRegisterDb(self._dbName, allowDuplicate=True)